lib/pbench/cli/agent/commands/tools/kill.py

```diff
@@ -107,12 +107,13 @@ def killem(self, echo: Callable[[str], None]) -> None:
                 echo(f"\t\terror killing {pid}: {exc}", err=True)
 
 
-def gen_run_directories(pbench_run: pathlib.Path) -> Iterable[Tuple[pathlib.Path, str]]:
-    """Generate the list of run directories available under ${pbench_run},
-    yielding a Path object for that directory, along with its recorded
-    UUID.
+def gen_result_tm_directories(
+    pbench_run: pathlib.Path,
+) -> Iterable[Tuple[pathlib.Path, str]]:
```

Comment:
Sometimes I really hate `black`. I'm tempted to suggest a `NamedTuple` or even an `@dataclass` to give your path+uuid a real type, which could make it easier to read and just might "incidentally" shorten this declaration enough to fit on a line❓ 

Anyway, it's a simple tuple and I'm not going to hold the PR over this, so I'll pre-resolve this and approve: but something to consider if you make another pass for something else.